roles/chaos-beaver/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: "create chaos directory"
+  file:
+    path: "{{chaos_home}}"
+    state: "directory"
+    owner: "{{chaos_user}}"
+    group: "{{chaos_group}}"
+    mode: "0754"
+
+- name: "copy shell script for killing container"
+  copy:
+    src: "{{container_kill_script}}"
+    dest: "{{chaos_home}}/{{container_kill_script}}"
+    owner: "{{chaos_user}}"
+    group: "{{chaos_group}}"
+    mode: "0754"
+
+- name: "generate jenkins kill script"
+  template:
+    src: "{{jenkins_kill_script}}.j2"
+    dest: "{{chaos_home}}/{{jenkins_kill_script}}"
+    owner: "{{chaos_user}}"
+    group: "{{chaos_group}}"
+    mode: "0554"
+
+- name: "check if line present in sudoers file"
+  lineinfile:
+    path: "/etc/sudoers"
+    line: "{{ chaos_user }} ALL=(ALL) NOPASSWD: {{container_kill_script}}"
```

Comment:
Should this line instead be..
```
"{{ chaos_user }} ALL=(ALL) NOPASSWD: {{chaos_home}}/{{container_kill_script}}"
```
Also, it might be good to put that line into a variable since it seems like it should be used twice.